dice.js

```diff
@@ -0,0 +1,16 @@
+const rollDice = function (){
+    const Dice1 = math.floor(math.random()*(6)+1);
+    const Dice2 = math.floor(math.random()*(6)+1);
+
+    return Dice1+Dice2
+};
+Console.log (rollDice());
+ // BOUNS!
+
+ const rollDice = function (more){
+     let result = null
+     for (let i =more; i>1; i--) {
+         let value = math.floor(math.random()*(10)+1);
```

Comment:
use the `result` value you declared to store the sum of the random dice numbers.
like this:
`result += math.floor(math.random()*(6)+1);`